src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.stories.tsx

```diff
@@ -40,22 +40,19 @@ TrialInstanceJustCreated.args = {
   expiryDate: addHours(new Date(), 48),
   instanceType: "eval",
```

Comment:
Unrelated to PR. Do you know what this does?

Reply:
expiryDate and instanceType are  using to show expiration message on details page for trial/eval kafka.
for example: `This trial instance will expire in 15 hours 34 minutes`